models/tornado_cash/tornado_cash_deposits.sql

```diff
@@ -0,0 +1,250 @@
+{{ config(
+        alias ='deposits',
+        materialized='incremental',
+        partition_by='block_date'
+        )
+}}
+
+SELECT *
+FROM
+(
+        -- Ethereum (ETH)
+        SELECT tc.evt_block_time AS block_time
+        , '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' AS currency_contract
+        , 'ETH' AS currency_symbol
+        , 'Ethereum' AS blockchain
+        , 'Classic' AS tornado_version
+        , et.from AS depositor
+        , tc.contract_address AS contract_address
+        , CASE WHEN tc.contract_address='0x12d66f87a04a9e220743712ce6d9bb1b5616b8fc' THEN 0.1
+                WHEN tc.contract_address='0x47ce0c6ed5b0ce3d3a51fdb1c52dc66a7c3c2936' THEN 1
+                WHEN tc.contract_address='0x910cbd523d972eb0a6f4cae4618ad62622b39dbf' THEN 10
+                WHEN tc.contract_address='0xa160cdab225685da1d56aa342ad8841c3b53f291' THEN 100
+                END AS amount
+        , tc.evt_tx_hash AS tx_hash
+        , tc.leafIndex AS leaf_index
+        , tc.evt_index
+        , date_trunc(tc.evt_block_time, 'day') as block_date
+        FROM {{ source('tornado_cash_ethereum','eth_evt_Deposit') }} tc
+        LEFT JOIN {{ source('ethereum','transactions_0006') }} et ON et.hash=tc.evt_tx_hash
```

Comment:
should have a second if is incremental, 'AND et.block_time >= (select max(block_time) from {{ this }})'
to be safe, double check the column name, assuming it drops the 'evt' preface